tinygrad/runtime/ops_metal.py

```diff
@@ -60,23 +61,26 @@ def transfer(self, dest:Any, src:Any, sz:int):
     encoder.endEncoding()
     command_buffer.commit()
     self.device.mtl_buffers_in_flight.append(command_buffer)
-  def from_buffer(self, src:memoryview) -> Optional[Any]:
-    ret = self.device.device.newBufferWithBytesNoCopy_length_options_deallocator_(src, len(src), Metal.MTLResourceStorageModeShared, None)
-    if ret: self.device.mv_in_metal.append(src)
-    return ret
   def _free(self, opaque:Any): opaque.release()
   def as_buffer(self, src:Any) -> memoryview:
     self.device.synchronize()
     return src.contents().as_buffer(src.length())
   def copyin(self, dest:Any, src:memoryview): self.as_buffer(dest)[:] = src
   def copyout(self, dest:memoryview, src:Any): dest[:] = self.as_buffer(src)
+  def load_buffer(self, dest:Any, src:Any, handles:Dict={}):
```

Comment:
While this is clever, this should be a property added to the ud buffer.

Reply:
no, they are not closed rn. i'll fix that if I see a speed increase of load_buffer